src/apps/search-header/search-header.test.ts

```diff
@@ -0,0 +1,213 @@
+                term: "Harry Potter og fangen fra Azkaban",
+                work: {
+                  workId: "work-of:870970-basis:22995154",
+                  titles: { main: ["Dummy Some Title"] },
+                  creators: [
+                    { display: "Dummy Jens Jensen" },
+                    { display: "Dummy Some Corporation" }
+                  ],
+                  manifestations: { first: { pid: "870970-basis:22995154" } }
+                }
+              },
+              {
+                type: "SUBJECT",
+                term: "Harry Dresden",
+                work: {
+                  workId: "work-of:870970-basis:53182623",
+                  titles: { main: ["Dummy Some Title"] },
+                  creators: [
+                    { display: "Dummy Jens Jensen" },
+                    { display: "Dummy Some Corporation" }
+                  ],
+                  manifestations: { first: { pid: "870970-basis:53182623" } }
+                }
+              },
+              {
+                type: "TITLE",
+                term: "Harry Potter og Fønixordenen",
+                work: {
+                  workId: "work-of:870970-basis:25245784",
+                  titles: { main: ["Dummy Some Title"] },
+                  creators: [
+                    { display: "Dummy Jens Jensen" },
+                    { display: "Dummy Some Corporation" }
+                  ],
+                  manifestations: { first: { pid: "870970-basis:25245784" } }
+                }
+              },
+              {
+                type: "CREATOR",
+                term: "Yuval Noah Harari",
+                work: {
+                  workId: "work-of:870970-basis:51725832",
+                  titles: { main: ["Dummy Some Title"] },
+                  creators: [
+                    { display: "Dummy Jens Jensen" },
+                    { display: "Dummy Some Corporation" }
+                  ],
+                  manifestations: { first: { pid: "870970-basis:51725832" } }
+                }
+              },
+              {
+                type: "TITLE",
+                term: "Harry Potter and the philosopher's stone",
+                work: {
+                  workId: "work-of:870970-basis:42042455",
+                  titles: { main: ["Dummy Some Title"] },
+                  creators: [
+                    { display: "Dummy Jens Jensen" },
+                    { display: "Dummy Some Corporation" }
+                  ],
+                  manifestations: { first: { pid: "870970-basis:42042455" } }
+                }
+              }
+            ]
+          }
+        }
+      }
+    });
+    cy.intercept("GET", "**/covers**", {
+      statusCode: 200,
+      body: [
+        {
+          id: "870970-basis:25245784",
+          type: "pid",
+          imageUrls: {
+            small: {
+              url: "https://res.cloudinary.com/dandigbib/image/upload/t_ddb_cover_small/v1543886150/bogportalen.dk/9788702029444.jpg",
```

Comment:
Instead of serving an external image you can also mock it like here:
https://github.com/danskernesdigitalebibliotek/dpl-react/pull/83/files#diff-2681eddd9ccddb0757dced492f1e67c5065af0d811d13c6d071a7b9d09124753R92

Reply:
Very good point! I am mocking the covers now too :)